SECURITY.md

```diff
@@ -0,0 +1,14 @@
+# Flatcar Security 
+<todo - describe a security Schedule of tracking security issues of Flatcar>
+- Describe the process of tracking security issues for Flatcar, especially tracking issues from upstream projects like Gentoo Linux.
```

Comment:
To keep Flatcar secure, the maintainers put a strong focus on tracking new and existing security issues.
Dealing with Security concerns is owned by the [Flatcar Security team](https://github.com/orgs/flatcar/teams/flatcar-security-team), a sub-set of the Maintainers team, and elected by the Maintainers (see [governance.md](./governance.md)).

While the team actively researches and tracks new and existing security issues, it may also be notified of issues via [security@flatcar-linux.org](mailto:security@flatcar-linux.org).

The Security team meets in a fortnightly cadence, in a private video call.
The team maintains an internal list of security Primaries and Secondaries, which are rotated on a weekly basis. 
Primary and Secondary are expected to actively engage in security work each day, including executing the Runbook (see below) and working on fixing ongoing security issues.

Undisclosed security issues are tracked in a private repository only accessible by members of the security team.
Public issues are tracked publicly in the project's main issue tracker.

Security issues are addressed by releasing an updated OS image. Releases may be expedited depending on the issues' severity. For each release, release notes contain a concise list of security issues fixed. Also, a separate, detailed report on each of the issues addressed is part of every release.